scripts/do-release.sh

```diff
@@ -10,6 +10,13 @@ set -o pipefail
 
 ### Functions: General ###
 
+# Run the supplied command in the background so that `errexit` and `pipefail` are honored.
```

Comment:
```suggestion
# Run the supplied command in the background so that `errexit` and `pipefail` are ignored.
```

Reply:
`pipefail` doesn’t matter anyway, because

    record_errors foo | bar

will record errors from `foo`, and the whole `record_errors foo` invocation will be piped; there’s no way to run

    record_errors 'foo | bar'

Reply:
Not that I mind changing documentation too much, but they were actually being ignored before this change, so having it as you suggest sounds confusing to me

Reply:
Right, it needs a little more explanation, since the settings are applied inside the command but don’t affect the `record_errors` function itself (*i.e.* a function called with `errexit` enabled will abort on error, but won’t caused `record_errors` to abort, thanks to the `||`). Given that running in the background specifically takes care of the former, we can leave the comment as-is.